sema/subyt/sources.py

```
@@ -115,8 +118,22 @@
             only the first record for each expanded value is processed as part
             of the source.
         @type unique_pattern: str | None
+        @param fake_empty: if True, any error in the factory process will lead
+            to returning a fake empty source. Else the error is raised.
         """
-        source = SourceFactory._make_core_source(identifier)
+        try:
+            source = SourceFactory._make_core_source(identifier)
+        except ValueError as e:
+            if not fake_empty:
+                raise e
```

Ruff (0.8.2)

Use `raise` without specifying exception name

Remove exception name

(TRY201)